app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt

```diff
@@ -1121,6 +1122,35 @@ class PinPasswordActivityTest {
     }
   }
 
+  @Test
+  fun testResetPinDialogFragmentPresenter_userNameMatches_isDisplayed() {
```

Comment:
The test name looks incorrect.

`testPinPassword_clickforgotPin_enterAdminPin_clickSubmit_dialogMessageIsCorrect`

Reply:
@rt4914  I have changed the test name. Please review.
Thanks.